packages/suite-base/src/components/MessagePathSyntax/MessagePathInput.tsx

```diff
@@ -192,7 +193,18 @@ export default React.memo<MessagePathInputBaseProps>(function MessagePathInput(
     [messagePathStructuresForDataype, noMultiSlices, topics, validTypes],
   );
 
+  const [currentPath, setCurrentPath] = useState<string>(path);
+  const [debouncedPath] = useDebounce(currentPath, 250, { maxWait: 250 });
+
   const onChangeProp = props.onChange;
+  useEffect(() => {
+    if (props.index == undefined) {
```

Comment:
Shouldnt it be === instead of ==?

Reply:
I'd say yes and initially I did it, but the lint asks to leave ==